nodejs/awsx/apigateway/README.md

```diff
@@ -0,0 +1,139 @@
+# Pulumi API Gateway Components
+
+Pulumi's API for simplifying working with [API Gateway](https://aws.amazon.com/api-gateway/). The API currently provides ways to define a route that accepts and forwards traffic to a specified destination.
+
+## Defining an Endpoint
+
+To define an endpoint you will either need to specify the routes or a swagger string. You can also define the stage name (else it will default to "stage"). A `stage` is an addressable instance of the Rest API.
+
+### Routes
+
+The destination is determined by the route, which can be an Event Handler Route, a Static Route, an Integration Route or a Raw Data Route.
+
+#### Event Handler Route
+
+An Event Handler Route is a route that will map to a [Lambda](https://aws.amazon.com/lambda/). You will need to specify the path, method and the lambda. Pulumi allows you to define the lambda inline with your Pulumi code & provisions that appropriate permissions.
+
```

Comment:
1. "with your app(lication) code"
2. "and" is generally preferred over "&"

Reply:
"provisions that appropriate permissions." 

Maybe "provisions the appropriate permissions on your behalf so that APIGateway can communicate with the Lambda."